lib/presentation/organisms/intro/tips_organism.dart

```diff
@@ -42,11 +44,13 @@ class TipsOrganism extends StatelessWidget {
           const SeparatorAtom(variant: SeparatorVariant.farApart),
           Center(
             child: Platform.isAndroid
-                ? ButtonAtom(
-                    variant: ButtonVariant.primary,
-                    onPressed: () => DndController.instance.toggleDnd(),
-                    text: 'dnd',
-                  )
+                ? dndEnable
+                    ? TextAtom('dnd_on')
+                    : ButtonAtom(
+                        variant: ButtonVariant.primary,
+                        onPressed: () => DndController.instance.toggleDnd(),
+                        text: 'Do Not Disturb',
+                      )
                 : const TextAtom('dnd_ios'),
```

Comment:
If we can dnd state on iOS the text should also view 'dnd_on' state

Reply:
Currently we cannot detect the state of dnd on ios [as mentioned in this answer](https://stackoverflow.com/a/17560797)

Reply:
Amazing Apple.
Ok thanks for the comment